x-pack/solutions/security/plugins/security_solution/public/detections/configurations/security_solution_detections/render_cell_value.tsx

```diff
@@ -84,22 +80,21 @@ export const CellValue = memo(function RenderCellValue({
       !isDetails,
     [columnId, isDetails, rowIndex, tableType]
   );
-  const { browserFields } = useSourcererDataView(sourcererScope);
-  const browserFieldsByName = useMemo(() => getAllFieldsByName(browserFields), [browserFields]);
-  const getTable = useMemo(() => dataTableSelectors.getTableByIdSelector(), []);
-  const license = useLicense();
-  const viewMode =
-    useDeepEqualSelector((state) => (getTable(state, tableId ?? '') ?? tableDefaults).viewMode) ??
-    tableDefaults.viewMode;
-
-  const gridColumns = useMemo(() => {
-    return getColumns(license);
-  }, [license]);
-
-  const columnHeaders = useMemo(() => {
-    return viewMode === VIEW_SELECTION.gridView ? gridColumns : eventRenderedViewColumns;
-  }, [gridColumns, viewMode]);
+  const cellValueContext = useContext(AlertTableCellContext);
 
+  if (!cellValueContext) {
+    const contextMissingError = new Error(
+      'render_cell_value.tsx: CellValue must be used within AlertTableCellContextProvider'
+    );
+
+    notifications.toasts.addError(contextMissingError, {
+      title: 'AlertTableCellContextProvider is missing',
+      toastMessage: 'CellValue must be used within AlertTableCellContextProvider',
+    });
+    throw new Error(contextMissingError.message);
```

Comment:
This will anyway call the error boundary and crash whatever there is on the screen. Not sure if adding error to notification helps.

Reply:
I think there is an error boundary in the table...IT actually just shows an error message in each cell 😅

Reply:
It ends up looking like this:

![image](https://github.com/user-attachments/assets/536d0677-deb3-4b3f-9dfd-bbaa563348b3)

Reply:
Beautiful 🤣